valhalla/mjolnir/landmark_builder.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <boost/property_tree/ptree.hpp>
+#include <string>
+#include <unordered_map>
+
+namespace valhalla {
+namespace mjolnir {
+static const std::string default_landmark_name = "";
+
+// obvious landmark types for vehicle routing
+enum class LandmarkType : uint8_t {
+  null = 0,
+  restaurant = 1,
+  school = 2,
+  fast_food = 3,
+  cafe = 4,
+  fuel = 5,
+  bank = 6,
+  pharmacy = 7,
+  kindergarten = 8,
+  bar = 9,
+  hospital = 10,
+  post_office = 11,
+  pub = 12,
+  clinic = 13,
+  police = 14,
+  fire_station = 15,
+  car_wash = 16,
+  charging_station = 17,
+  college = 18,
+  university = 19,
```

Comment:
these things are probably too large to use to navigate, id drop them

Reply:
make sense